src/Administration/Resources/app/administration/src/module/sw-product/snippet/en-GB.json

```
@@ -93,7 +93,7 @@
       "placeholderMaxPurchase": "Max. amount purchaseable...",
       "placeholderPurchaseSteps": "Enter purchase steps...",
       "placeholderRestockTime": "Enter restock time in days...",
-      "placeholderEan": "Enter EAN number...",
+      "placeholderEan": "Enter GTIN number...",
```

```suggestion
      "placeholderEan": "Enter GTIN...",
```
GTIN stays for `Global Trade Item **Number**`

---

Done